shopfloor/actions/message.py

```diff
@@ -818,8 +818,10 @@ def unable_to_pick_qty(self):
     def lot_replaced_by_lot(self, old_lot, new_lot):
         return {
             "message_type": "success",
-            "body": _("Lot %(old_lot_name)s replaced by lot %(new_lot_name)s.").format(
-                old_lot.name, new_lot.name
+            "body": _(
```

Comment:
right below this fix there's another message that should be fixed for the same reason: `package_replaced_by_package`. Please check if there's any other message that is misusing formatting w/ `%s` w/ `.format`.